test/replication-luatest/gh_6842_qsync_applier_order_test.lua

```diff
@@ -95,6 +95,30 @@ local function wait_synchro_owner(server, owner_id)
     end, {owner_id})
 end
 
+--
+-- Wait until the server sees synchro queue owner as the given ID.
```

Comment:
Stale comment

Reply:
Thanks, changed to `Wait until the server sees synchro queue has the given length.`.